tests/chainer_tests/links_tests/model_tests/test_vision.py

```diff
@@ -0,0 +1,220 @@
+import unittest
+import warnings
+
+import numpy
+
+from chainer import cuda
+from chainer.links.model.vision import resnet
+from chainer.links.model.vision import vgg
+from chainer import testing
+from chainer.testing import attr
+from chainer.variable import Variable
+
+
+@unittest.skipUnless(resnet.available, 'Pillow is required')
+class TestResNet50Layers(unittest.TestCase):
+
+    def setUp(self):
+        self.link = resnet.ResNet50Layers(pretrained_model=None)
+
+        # Suppress warning that arises from zero division.
+        warnings.filterwarnings('ignore', category=RuntimeWarning)
+
+    def test_available_layers(self):
+        result = self.link.available_layers
+        self.assertIsInstance(result, list)
+        self.assertIsNot(len(result), 0)
```

Comment:
The list of available_layers for ResNet is known and fixed. Can't we use the length directly here (9 layers, if I'm not mistaken?)